lib/commands.js

```diff
@@ -273,10 +274,26 @@ var operations = module.exports.operations = {
       context.newTemplate = templateBody;
       context.oldTemplate = oldTemplate;
       context.oldParameters = oldParams;
+
       context.next();
     });
   },
 
+  getMasterConfig: function(context) {
+    if (context.overrides.masterConfig) {
```

Comment:
One way to avoid the dreaded Too Much Indenting In My JavaScript problem is to use a conditional like this to break you out of the function right away. So instead of 

```js
if (context.overrides.masterConfig) {
  do one thing
} else {
  do some other thing
}
```

... you can

```js
if (!context.overrides.masterConfig) return context.next();

do the thing that requires masterConfig
```